src/Components/Results/Tabs/Tabs.tsx

```diff
@@ -7,11 +10,27 @@ type ResultsTabsProps = {
 const ResultsTabs = ({ currentTab }: ResultsTabsProps) => {
```

Comment:
We can remove the current tab prop because we are not using it. You can also get rid of the type.